cni/pkg/plugin/iptables.go

```diff
@@ -57,10 +57,9 @@ func (ipt *iptables) Program(netns string, rdrct *Redirect) error {
 	log.Infof("nsenter args: %s", strings.Join(nsenterArgs, " "))
 	out, err := exec.Command("nsenter", nsenterArgs...).CombinedOutput()
 	if err != nil {
-		log.WithLabels("err", err, "out", out).Errorf("nsenter failed ")
-		log.Infof("nsenter out: %s", out)
+		log.Errorf("nsenter failed %v, with output %v", err, out)
```

Comment:
Looks like you will have to plumb log context all the way thru from cni, otherwise pod name and other params will not be available.